src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap

```diff
@@ -1,32 +1,32 @@
 ---
-created: "2019-04-26T02:30:09.035774600Z"
+created: "2019-05-05T19:54:12.487950400Z"
 creator: insta@0.7.4
 source: src/bin/snapshots.rs
 expression: forest
 ---
 Ok(
-    _ => ModuleContents {
-        attrs: _ => [],
-        items: _ => [
-            _ => Item {
-                attrs: _ => [],
-                kind: _ => ItemKind::Fn {
-                    header: _ => FnHeader,
-                    decl: _ => FnDecl {
+    ModuleContents {
+        attrs: [],
+        items: [
+            Item {
+                attrs: [],
+                kind: ItemKind::Fn {
+                    header: FnHeader,
+                    decl: FnDecl {
                         name: _
                     },
-                    body: _ => Block {
-                        attrs: _ => [],
-                        stmts: _ => [
-                            _ => Stmt::Expr(
-                                _ => Expr {
-                                    attrs: _ => [],
-                                    kind: _ => ExprKind::Path(
-                                        _ => QPath::Unqualified(
-                                            _ => Path {
-                                                path: _ => RelativePath {
-                                                    segments: _ => [
-                                                        _ => PathSegment {
+                    body: Block {
+                        attrs: [],
+                        stmts: [
+                            Stmt::Expr(
+                                Expr {
+                                    attrs: [],
+                                    kind: ExprKind::Path(
+                                        QPath::Unqualified(
+                                            Path {
+                                                path: RelativePath {
+                                                    segments: [
+                                                        PathSegment {
                                                             ident: _
```

Comment:
Hmm; we are not showing `IDENT`s -- would be nice to fix in a different PR?

Reply:
This is a gll issue (cc @eddyb): the debug representation is `ident: Span..Span` with the `proc_macro2` shim.